<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WAxisSelector</class>
 <widget class="QDialog" name="WAxisSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="in_name">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="c_axis">
     <item>
      <property name="text">
       <string>time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>angle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>axial direction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>frequency</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wavenumber</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="c_unit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>s</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="b_filter">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
